.github/workflows/ci.yml

```
@@ -0,0 +1,55 @@
+name: Test Status
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    name: Test Code
+    env:
+      CI: true
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [16, 18, 20]
+        experimental: [false]
+        include:
+          - os: ubuntu-latest
+            node-version: latest
+            experimental: true
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }} on ${{ matrix.os }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - run: npm test
+      - run: npm update
+      - run: npm ci
+      - run: npm test
+      - name: Codecov
+        uses: codecov/codecov-action@v3.1.4
+        with:
+          verbose: true
```

Just configured the `secret.CODECOV_TOKEN` which needs to be added here, right?
```suggestion
      - name: Codecov
        uses: codecov/codecov-action@v3.1.4
        with:
          verbose: true
        env:
          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

---

It shouldn't be needed because it will look for the secret if it is named correctly. I don't have that line in any of my other repos.

---

I'll comment this out for now to see if we can get the other stuff running.

---

Ok, fixed it. See the report at the bottom of the PR. It didn't need the `env` added here, it needed a workflow file in the `main` branch to get things rolling.